patches/lua-tree.patch

```diff
@@ -61,7 +61,7 @@ tools/utils.lua:
 - Use patched.url intead of socket.url
 - Use Lua 5.1 based xpcall
 - Use go.ipmatcher instead of resty.ipmatcher
-- Use go.re2 instead of ngx.re
+- Use gluare instead of ngx.re
```

Comment:
Let's remove these 3 comments and before the first at the top of the patch we can add a blurb about ngx.re.<find|match> being replaced with gluare.

```
ngx.re.find and ngx.re.match have been overloaded to use gluare::find and gluare::match. gluare::find uses RE2 which is not a direct replacement for PCRE so some modification to Lua source code may still be required in order to make functionality work appropriately.
```

Something along those lines; please rephrase to make better.